x/provider/ante/ante_test.go

```diff
@@ -0,0 +1,151 @@
+package ante
```

Comment:
I think this can be in `ante_test` - will help with preventing circular imports.